dynawo/sources/Common/Dictionaries/DYNError_en_GB.dic

```diff
@@ -52,8 +52,8 @@ SolverUnstableZMode         =             discrete events are unstable during th
 SolverUnbalanced            =             the number of algebraic/differential variables is different from the number of algebraic/differential equations in the simulated problem
 WrongLinearSolverChoice     =             the linear solver name provided is not valid
 SolverMissingParam          =             no value found for mandatory solver parameter '%1%' (parameter set '%2%', parameter file '%3%')
-SolverJacobianWithNulColumn =             jacobian has nul column %1%: equation '%2%' is ill-formed
-SolverJacobianWithNulRow    =             jacobian has nul row %1%: variable '%2%' is ill-formed
+SolverJacobianWithNulColumn =             jacobian has null column %1%: equation '%2%' is ill-formed
```

Comment:
we are speaking a column with only zero here, so let's use zero instead of null here

Reply:
Corrected.